Archipelago.MultiClient.Net.Analyzers/Archipelago.MultiClient.Net.Analyzers/Analyzers/ItemFlagsComparisonInSwitchCaseDiagnostics.cs

```diff
@@ -47,12 +47,15 @@ private void AnalyzeSwitch(SyntaxNodeAnalysisContext context)
                         TypeInfo typeInfo = context.SemanticModel.GetTypeInfo(identifier);
                         if (ArchipelagoTypeUtils.IsTypeItemFlags(typeInfo.Type, context.Compilation))
                         {
-                            context.ReportDiagnostic(Diagnostic.Create(NoItemFlagsComparisonsInSwitchCaseConstants, label.GetLocation()));
+                            // Check if the identifier is ItemFlags.None
+                            if (identifier.Name.ToString() != "None")
```

Comment:
I think the better way to do this is by checking the actual integer value of the enum is 0, I do this in the if statement version for reference. That would make it more resilient to renames

Reply:
I was able to make this happen with `SemanticModel.GetConstantValue()`, so that's nice.